spec/models/unaccompanied_minor_output_spec.rb

```diff
@@ -0,0 +1,178 @@
+require "rails_helper"
+FULL_ATTRIBUTES_LIST = %i[
+  reference
```

Comment:
```suggestion
adult_number
adults_at_address
certificate_reference
created_at
different_address
email
family_name
given_name
has_other_names
has_other_nationalities
have_parental_consent
identification_number
identification_type
ip_address
is_born_after_december
is_committed
is_consent
is_living_december
is_permitted
is_unaccompanied
is_under_18
minor_contact_details
minor_contact_type
minor_date_of_birth
minor_date_of_birth_as_string
minor_email
minor_family_name
minor_given_name
minor_phone_number
nationality
no_identification_reason
other_adults_address
other_family_name
other_given_name
other_names
other_nationalities
other_nationality
phone_number
privacy_statement_confirm
reference
residential_line_1
residential_line_2
residential_postcode
residential_town
sponsor_address_line_1
sponsor_address_line_2
sponsor_address_postcode
sponsor_address_town
sponsor_date_of_birth
sponsor_date_of_birth_as_string
sponsor_declaration
started_at
type
uk_parental_consent_filename
uk_parental_consent_file_size
uk_parental_consent_file_type
uk_parental_consent_file_uploaded_timestamp
uk_parental_consent_file_upload_rid
uk_parental_consent_saved_filename
ukraine_parental_consent_filename
ukraine_parental_consent_file_size
ukraine_parental_consent_file_type
ukraine_parental_consent_file_uploaded_timestamp
ukraine_parental_consent_file_upload_rid
ukraine_parental_consent_saved_filename
user_agent
version
```

Reply:
Sorted for readability ?

Reply:
it keeps the same order as in `UnaccompaniedMinor` model so, if something changes, it's a 1-to-1 match to find it